packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 47.1.0 - October 12, 2019
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Fix for overwrite label ([PR #1704](https://github.com/forcedotcom/salesforcedx-vscode/pull/1704))
```

Comment:
I can add detail about what label specifically was updated if needed

Reply:
For minor fixes we tend to use `- We fixed some minor under-the-hood bugs.` without linking to specific PRs